task_2.py

```diff
@@ -0,0 +1,12 @@
+"""Introduction to programming’: Task 2,
+   Kyshynska Taisiia"""
+
+print("""Introduction to programming’: Task 2
+         Kyshynska Taisiia""")
+
+x = int(input('x = '))
```

Comment:
you need to add validations for a case when user enter some str that cannot be converted to int
for ex. " Kyshynska Taisiia"

Reply:
As well, you use the same pattern 3 times. 
You can move it to a constant template and format it with variable name